guides/_posts/2014-01-11-ifelse.html

```diff
@@ -36,73 +49,83 @@
 
 {% endhighlight %}
 
-But, any property can be used within an `@if`, `@else` conditional.  So, if you want a layout to be responsive to a panel's width:
+But, any property can be used within an `@if`, `@else` conditional.  So, if you want a layout to be responsive to a section's width:
 
 {% highlight css %}
 
-/* GSS: */
-@if #panel[width] >= 800 {
-  /* ... */
-}
-@else #panel[width] >= 500 {
-  /* ... */
+@if section[width] <= 400 {
+  section {
+    article {
+      :first[top] == ^[top];
+      bottom: == :next[top];
+      left: == ^[left];
+      width: == ^[width];
+      height: == ^[height] / 2;
+    }
+  }
 }
-@else {
-  /* ... */
+@else  {
+  section {
+    article {
+      width: == ^[width] / 2;
+      height: == ^[height];
+      top: == :next[top];
+      right: == :next[left];
+    }
+  }
 }
```

Comment:
I think this could be simplified a lot to remove other concepts and put the most basic point across.

Something like this:

```
section {
  article {
    @if ^[width] <= 400 {
      width: == ^[width];
      height: == ^[height] / 2;
    }
    @else  {
      width: == ^[width] / 2;
      height: == ^[height];
    }
  }
}
```